src/Core/Managed/Net45/Extensibility/Implementation/External/UserContextData.cs

```diff
@@ -77,6 +77,15 @@ public string StoreRegion
         {
             get { return this.tags.GetTagValueOrNull(ContextTagKeys.Keys.UserStoreRegion); }
             set { this.tags.SetStringValueOrRemove(ContextTagKeys.Keys.UserStoreRegion, value); }
+        }
+
+        /// <summary>
+        /// Gets or sets the AuthUserId of an application-defined account associated with the user.
+        /// </summary>
+        public string AuthUserId

```

Comment:
This is the name of the field in the BOND schema. I can't/shouldnt modify it here